lib/usdUfe/utils/Utils.cpp

```diff
@@ -216,6 +216,9 @@ std::string prettifyName(const std::string& name)
         { "usd", "USD" },
         { "mtlx", "MaterialX" },
         { "lookdevx", "LookdevX" },
+        { "Ui Nodegraph Node", "Ui" },
+        { "Translate Scale Pivot", "Scale Pivot" },
+        { "Translate Rotate Pivot", "Rotate Pivot" },
```

Comment:
Wouldn't the compensation ones still have a very long name and get trimmed?

Reply:
No, it gets fixed by the same replacement as the pivot one. I tested it manually to be sure :)